modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc

```diff
@@ -62,3 +47,17 @@ When "_{secure-view-label}_" is enabled, any attempts to download the file will
 Additionally, select, copy, and paste are disabled.
 
 image:enterprise/collaboration/access-denied.png[Access denied to a document when it is protected by Secure View]
+
+== Supported File Formats
+
+Secure View only supports a limited number of file formats; these are:
+
+* Microsoft Word (.docx)
+* Microsoft Excel (.xlsx)
+* Microsoft PowerPoint (.pptx)
+* OpenDocument Text Document (.odt)
+* OpenDocument Presentation Document (.odp)
+* OpenDocument Spreadsheet Document (.ods)
+* PDF
+
+If a folder shared with Secure View contains unsupported file types (e.g., JPG), they will not be accessible in any way.
```

Comment:
How about removing "in any way" at the end, as "will not be accessible" should be sufficiently explicit?

Reply:
again, you wrote this, the only change I made was in the prerequisites.

Reply:
Thanks.